content/en-us/reference/engine/enums/RollOffMode.yaml

```diff
@@ -38,6 +43,8 @@ items:
       A hybrid model which follows the `Inverse` model when close to
       `Class.Sound.RollOffMinDistance` and the `LinearSquare` model when close
       to `Class.Sound.RollOffMaxDistance`.
+      This is done through `Libraries.math.min`((`Class.Sound.RollOffMinDistance`/`distance`)*`LinearSquare`),
```

Comment:
Here, do you mean a `,` instead of a `*`? As written with the `*`, this `math.min()` example only contains one value to use as the minimum...

Reply:
Sorry, when I was porting over the code, I must've misinterpreted what math.min did. Removing math.min entirely would be best

Reply:
```
(`Class.Sound.RollOffMinDistance`/`distance`)*`LinearSquare`)
```